handler/setup.go

```
@@ -34,73 +34,44 @@ func (h *SetupHandler) Setup(c echo.Context) error {
 
-		s.Title = title
-		s.Description = desc
+		s.Title = form.Title
+		s.Description = form.Description
+
+		user := &domain.User{
+			ID:       snowflake.Generate(),
+			Username: form.Username,
+			Role:     domain.Admin,
+		}
+		user.SetPassword(form.Password1)
```

golangci-lint (1.62.2)

Error return value of `user.SetPassword` is not checked

(errcheck)